assets/css/sensei-course-theme/blocks/lesson-video.scss

```diff
@@ -0,0 +1,14 @@
+.sensei-course-theme-lesson-video {
+
+  .wp-block-embed, .wp-block-video {
+    margin: 0;
+  }
+  iframe, video {
+    width: 100%;
+    height: 100%;
+    position: absolute;
```

Comment:
I think absolute position won't work for most cases, the video needs to affect the layout or the next content would just flow under it. I added a new issue because the sizing seems to be a bigger problem: https://github.com/Automattic/sensei/issues/5769
We can merge this PR though and figure out a fix there.